packages/jest-runner/config.ts

```diff
@@ -59,6 +59,7 @@ export function loadConfig(optionsKey = "jazzerjs"): Options {
 	// Switch to fuzzing mode if environment variable `JAZZER_FUZZ` is set.
 	if (process.env.JAZZER_FUZZ) {
 		config.mode = "fuzzing";
+		config.dryRun = false;
```

Comment:
This should revert the instrumentation issue for now. Let's investigate applying bug detector instrumentation in regression mode in a separate PR.